src/Raven.Server/Documents/Patch/ScriptRunner.cs

```diff
@@ -103,7 +103,7 @@ public void AddTimeSeriesDeclaration(DeclaredFunction func)
             TimeSeriesDeclaration.Add(func.Name, func);
         }
 
-        public ReturnRun GetRunner(out SingleRun run)
+        public ReturnRun GetRunner(out SingleRun run, bool ignoreValidationErrors)
```

Comment:
Since `ignoreValidationErrors` isn't optional here can it be the first argument? Typically `out` is at the end

Reply:
Done.